<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1372</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </item>
   <item row="2" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Corrigido</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Descrição</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Usuário</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Data/Hora</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tipo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Versão QGIS</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Versão Plugins</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Sistema Operacional</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>MAC</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Início:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="startDe">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Fim:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="endDe">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadDataBtn">
       <property name="text">
        <string>Carregar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="showFixedCbx">
     <property name="text">
      <string>Mostrar Corrigidos</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="fixSelectionBtn">
       <property name="text">
        <string>Marcar selecionados como corrigidos</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deselectBtn">
       <property name="text">
        <string>Desselecionar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
